uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/modules/apis.js

```diff
@@ -0,0 +1,10 @@
+define([
+    'common/angular',
+    'common/modules/apis',
+], function (angular) {
+    'use strict';
+
+    return angular.module('leave-absences.apis', [
+        'common.apis',
```

Comment:
Also, is it really necessary to have this on a new line? (applies for all the other modules as well)

```js
return angular.module('leave-absences.apis', ['common.apis']);
```
If yes, then please use 2-spaced indentation